services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/EvaluationManager.java

```diff
@@ -91,4 +91,6 @@ public void deleteEvaluation(UserInfo userInfo, String id)
 	EvaluationRound getEvaluationRound(UserInfo userInfo, String evaluationId, String evaluationRoundId);
 
 	EvaluationRoundListResponse getAllEvaluationRounds(UserInfo userInfo, String evaluationId, EvaluationRoundListRequest request);
+
+	void adminMigrateSubmissionQuota(UserInfo userInfo);
```

Comment:
Instead of migrating all the existing evaluations as admins, we could instead implement in the portal a detection evaluations that still use the old submissionQuota and propose a migration path directly to the user: e.g. a button that allows the creator/manager of the evaluation to migrate to the new round version (e.g. a popup). In this case we do not risk to mess up with the user work. We can monitor the old evaluations over time and we could migrate as admins old (closed?) evaluations only.